lib/backports/2.7.0/enumerable/tally.rb

```diff
@@ -0,0 +1,11 @@
+unless Enumerable.method_defined? :tally
+  module Enumerable
+    def tally
+      # NB: By spec, we can't use Hash.new(0), because tally should return default-less hash
```

Comment:
We can, just need to clear it, say with `tap{|h| h.default = nil}`. Would be faster.